packages/modular-scripts/package.json

```diff
@@ -36,6 +36,7 @@
     "@rollup/plugin-node-resolve": "13.3.0",
     "@svgr/core": "6.2.1",
     "@svgr/webpack": "6.2.1",
+    "@types/lodash": "^4.14.191",
```

Comment:
Not sure in the slightest why that popped up now but it was complaining for missing typings for lodash